gradle/dependencies.gradle

```diff
@@ -1,40 +1,28 @@
 def versions = [
   'antlr': '3.4',
-  'assertj': '3.19.0',
-  'jsr305': '3.0.2',
   'jline': '0.9.94',
-  'gson': '2.9.0',
-  'slf4j': '1.7.25',
   'kryo': '2.22',
   'testing': '6.4',
-  'coral-calcite': '0.0.3',
   'hive': '1.2.2',
   'hadoop': '2.7.0',
   'pig': '0.15.0',
   'ivy': '2.4.0',
   'spark': '2.4.0',
   'spark3': '3.1.1',
-  'presto-parser': '0.262'
+  'linkedin-calcite-core': '1.21.0.151'
```

Comment:
nit: could you please add it in alphabetical order?

Reply:
Thanks for point it out, modified.